macros/filter_exceptions.sql

```diff
@@ -1,22 +1,28 @@
 {% macro filter_exceptions(model_ref) -%}
-  {{ return(adapter.dispatch('filter_exceptions', 'dbt_project_evaluator')(model_ref)) }}
+    {{ return(adapter.dispatch('filter_exceptions', 'dbt_project_evaluator')(model_ref)) }}
 {%- endmacro %}
 
 {% macro default__filter_exceptions(model_ref) %}
 
-{% set query_filters %}
-select
-    column_name,
-    id_to_exclude
-from {{ ref('dbt_project_evaluator_exceptions') }}
-where fct_name = '{{ model_ref.name }}'
-{% endset %}
+    {% set seed_exists = load_relation(ref('dbt_project_evaluator_exceptions')) is not none %}
```

Comment:
```suggestion
    {% set seed_exists = adapter.load_relation(ref('dbt_project_evaluator_exceptions')) is not none %}
```

maybe?

Reply:
check which package this relation is from?